src/app/Library/CrudPanel/Traits/Create.php

```diff
@@ -119,8 +119,13 @@ private function createRelationsForItem($item, $formattedRelations)
                     break;
                 case 'BelongsToMany':
                 case 'MorphToMany':
-                    $values = (isset($relationDetails['values'][$relationMethod]) && !empty($relationDetails['values'][$relationMethod])) ? $relationDetails['values'][$relationMethod] : [];
+                    $values = $relationDetails['values'][$relationMethod] ?? [];
                     $values = is_string($values) ? json_decode($values, true) : $values;
```

Comment:
```suggestion
$values = empty($values) ?  [] : (is_string($values) ? json_decode($values, true) : $values);
```

Reply:
I am now thinking that this may not be that dificult to read and we would avoid re-setting the variable. 
@tabacitu what do you think ? Is this readable enough ? Or should we opt by the first proposed solution ?

Reply:
Hmmm I liked the first one better, tbh

Reply:
@tabacitu I can reapply the first one again if you want

Reply:
Ah ok - didn't know there's three 😀

I like is_string() ? json_decode() : $values

Short & clear.